src/sys_rng.rs

```diff
@@ -52,3 +50,49 @@ impl TryRngCore for SysRng {
 }
 
 impl TryCryptoRng for SysRng {}
+
+/// A potentially-panicking [`RngCore`] interface over the system's preferred random number source.
+///
+/// This is a zero-sized struct. It can be freely constructed with just `UnwrappingSysRng`.
+///
+/// If possible, we recommend to use [`SysRng`] instead and to properly handle potential errors.
+///
+/// This struct is also available as [`rand::rngs::UnwrappingSysRng`] when using [`rand`].
+///
+/// # Usage example
+///
+/// `UnwrappingSysRng` implements [`RngCore`]:
+/// ```
+/// use getrandom::{rand_core::RngCore, UnwrappingSysRng};
```

Comment:
Nit: rustdoc formats this as:
```suggestion
/// use getrandom::{UnwrappingSysRng, rand_core::RngCore};
```